install/install.md

```diff
@@ -17,7 +17,7 @@ RAPIDS has several methods for installation, depending on the preferred environm
 **[2A. Setup Environment](#setup)** <br/>
 **[2B. Setup WSL2 Environment](#WSL2)** <br/>
 **[3A. Install RAPIDS (Release Selector)](#selector)** <br/>
-**[3B. Install RAPIDS (pip)](#pip)** <br/>
+**[3B. Install RAPIDS (pip)](#pip-install)** <br/>
```

Comment:
If pip is now part of the release selector, why do we need "3A Selector" and "3B pip"?

Reply:
Maybe the pip prerequisites should be in Section 2A "pip" instead of Section 3B?

Reply:
Also "setup" is two words if you don't mind adjusting that... https://writingexplained.org/setup-vs-set-up-difference

Reply:
If I can have one more request on this comment thread... can we use a nested structure for this "table of contents"? Something that looks more like:

![image](https://user-images.githubusercontent.com/3943761/233702969-dc05d4f8-d9a7-45b7-b660-b15fb3ef1422.png)

Reply:
@jarmak-nv We can defer some or all of these ideas to a subsequent PR. I'm happy for this PR to just fix the link.

Reply:
Totally agree with these changes, but let's push them over to align with the work in https://github.com/rapidsai/rapids.ai/issues/306 

I'll collect these comments and put them into that issue.

I've updated the setup wording since it's a no-discussion required fix, but the rest will take a bit of back and forth I imagine.

Reply:
Moved these comments over. I think we can resolve this conversation, but leave it to you @bdice if you have followups.

Reply:
Sweet! I appreciate that. I had seen that issue but forgot about it.